proto/viam/app/v1/billing.proto

```diff
@@ -71,6 +71,8 @@ message GetCurrentMonthUsageResponse {
   double discount_amount = 8;
   double total_usage_with_discount = 9;
   double total_usage_without_discount = 10;
+  double monthly_machines_usage_cost = 11;
```

Comment:
"monthly" is redundant since "Month" is in the message name

nit: "machine_usage_cost" can be vague as it can mean the sum of above costs. something like "total_per_machine_usage_cost" is clearer to me. I don't feel strongly about this.

Reply:
the "total" is a bit confusing because the other fields that start with "total" represent the full total cost whereas this could be one of many SKUs in the future potentially (although i know right now its the only sku applicable for the billing tier with it). made it `per_machine_usage_cost` instead?